Doc/howto/argparse.rst

```diff
@@ -646,7 +646,7 @@ to display *more* text instead::
    answer = args.x**args.y
    if args.verbosity >= 2:
        print("Running '{}'".format(__file__))
-   if args.verbosity >= 1:
+   if args.verbosity == 1:
```

Comment:
This example must not be changed; it is explicitly about showing how to use "verbosity level to display *more* text"

Reply:
@zware : ok, now I see the difference in output (filename) compared to previous examples. Agreed.